.cirrus.yml

```diff
@@ -0,0 +1,54 @@
+check_android_task:
+  name: Run Android instrumented tests
+  env:
+    SCREENSHOTS_ROOT_DIR: /data/data/com.kaspersky.kaspressample/files/screenshots
+    TUTORIAL_SCREENSHOTS_ROOT_DIR: /data/data/com.kaspersky.kaspresso.tutorial/files/screenshots
+    ALLURE_SAMPLE_SCREENSHOTS_ROOT_DIR: /data/data/com.kaspersky.kaspresso.tutorial/files/screenshots
+    ALLURE_SAMPLE_FAIL_SCREENSHOTS_ROOT_DIR: /data/user/0/com.kaspersky.kaspresso.alluresupport.sample/files/screenshots
+  only_if: $CIRRUS_PR != '' && $CIRRUS_BASE_BRANCH == 'master'
+
+  container:
+    image: cirrusci/android-sdk:30
```

Comment:
Why not 31?

Reply:
30 is the latest image provided by cirrus devs

Reply:
I think it's not critical =)